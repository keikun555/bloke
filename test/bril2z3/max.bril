@main(n: int, m: int) : int {
  condi: bool = lt n m;
  br condi .less .biggereq;
.less:
  ret m;
.biggereq:
  ret n;
}
