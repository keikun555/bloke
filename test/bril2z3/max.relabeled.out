@main(n: int, m: int): int {
  condi.0: bool = lt n.0 m.0;
  br condi.0 .less .biggereq;
.less:
  ret m.0;
.biggereq:
  ret n.0;
}
